node_modules/oae-core/manageaccess/manageaccess.html

```diff
@@ -24,7 +24,11 @@ <h3 id="manageaccess-modal-title">__MSG__MANAGE_ACCESS__</h3>
                             <span id="manageaccess-overview-shared-title"><!-- --></span>
                         </h4>
                         <div id="manageaccess-overview-shared" class="well">
-                            <ul id="manageaccess-overview-selected" class="oae-list oae-list-compact"><!-- --></ul>
+                            <ul id="manageaccess-overview-selected-shared-container" class="oae-list oae-list-compact manageaccess-overview-selected"><!-- --></ul>
+                        </div>
+                        <div id="manageaccess-overview-invitations" class="well hide">
+                            <div class="manageaccess-overview-header">__MSG__PENDING_INVITATIONS__</div>
+                            <ul id="manageaccess-overview-selected-invitations-container" class="oae-list oae-list-compact manageaccess-overview-selected"><!-- --></ul>
```

Comment:
The size of the icon can be reduced to 20px and the icon is not fully vertically center aligned